app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
```

Comment:
Given that it's tied to a specific user's `second_factor_locked_at`, I would say "yes", but I could be convinced otherwise if you preferred a separate class to handle this, but I don't think `GuestDecorator` is the right name because "guest" implies a visitor we can't identify, whereas we know exactly who this user is.